collections/_article/dev-snapshot-godot-3-7-alpha-1.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Dev snapshot: Godot 3.7 alpha 1"
+excerpt: "Since Godot 3.6's release in September 2024, we have been working hard on the new feature branch, 3.7."
+categories: ["release"]
+author: lawnjelly
+image: /storage/blog/covers/maintenance-release-godot-3-6-1.webp
+image_caption_title: CraftCraft
+image_caption_description: A game by Placeholder Gameworks
+date: 2025-06-25 12:00:00
+---
+
+Although the vast majority of contributors are busily working on Godot 4, a small problematic group refuse to stop improving Godot 3. Since [Godot 3.6](/article/godot-3-6-finally-released)'s release in September 2024, we have been working hard on the new feature branch, 3.7.
+
+Although a number of features are still in preparation, there have already been a number of improvements since 3.6, so consider this a sneak peak at some of the advances that will be available.
+
+## Downloads
+
+{% include articles/download_card.html version="3.7" release="alpha1" article=page %}
+**Standard build** includes support for GDScript, GDNative, and VisualScript.
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript, GDNative, and VisualScript.
+
+{% include articles/prerelease_notice.html %}
+
+As usual, you can try it live with the [**online version of the Godot editor**](https://editor.godotengine.org/releases/3.7.alpha1/) updated for this release.
+
+## Highlights
+
+#### Core
+
+Thanks to work particularly by [lawnjelly](https://github.com/lawnjelly) and [Lukas Tenbrink](https://github.com/Ivorforce), much of core has been heavily optimized, including faster access to `ProjectSettings`, faster object casting, move semantics, and use of faster data structures benefiting 2D and 3D.
+
+#### 3D physics interpolation
+
+In tandem with work on Godot 4.5, the 3D physics interpolation has been rewritten from the ground up ([GH-103685](https://github.com/godotengine/godot/pull/103685)).
+
+This has enabled us to fix a number of longstanding bugs and usability issues (particularly with non-interpolated tree branches), as well as more accurately representing pivot relationships which should give a smoother result.
+
+Luckily we have already worked through a lot of the bugs in 4.5 betas, but let us know if you spot any!
+
+#### Mutable audio drivers
+
+While most of you will be making games with constant audio, those making applications (as well as users of the Editor itself) can now benefit from experimental "mutable audio" via hotswapping the audio driver ([GH-63458](https://github.com/godotengine/godot/pull/63458)). This can dramatically reduce CPU usage. Audio is surprisingly one of the biggest CPU hogs in the Editor, despite it rarely playing any sounds!
+
+This is good news especially for those of you using low power PCs, or developing on battery devices (mobile / tablet / laptop). CPU use in the Editor (in conjunction with "vital updates only" mode) is now approximately 50x less than Godot 4.
```

Comment:
Do you really mean "50x less", as in CPU use in Godot 3.7 editor is about 2% of that of Godot 4? Or did you mean "50% less"?

Reply:
Wow, impressive! It seems like something should be forward-ported to 4.5 to improve that.

I think it would be better to compare with 3.6. People move up in the 3.x series, not down from 4.x to 3.x, I imagine.